tests/test_gh_workflows.py

```diff
@@ -22,6 +22,8 @@ class TestWorkflows(unittest.TestCase):
 
     def test_matrix_to_lock_version_defaults(self):
         """Test that the default versions in the lock-versions workflow mirror those from the schema-map."""
+        # TODO: when adding test exemption, this should be avoided for non Elastic by default since packages may
```

Comment:
You still plan on adding this in this PR?

Reply:
Was this intentionally removed?

Reply:
I removed it because, by default, if `test_only` is defined with no tests, then all are skipped. And if not, can `bypass`

(Well it was supposed to, but something changed, so I fixed it in https://github.com/elastic/detection-rules/pull/3407/commits/a84aef12c9c695ac1bd44533b6036ab3fef7748b)